engine-jupyter/templates/deployment.yaml

```diff
@@ -15,18 +15,31 @@ spec:
         app: {{ template "name" . }}
         release: {{ .Release.Name }}
     spec:
+      volumes:
+        - name: notebooks-dir
```

Comment:
I'd give this as an option as it limits how this is deployed. And it closes the door to use this with persistent volume claims which is the standard way of describing volumes in k8s. I'd write something like:

```
# TODO: Add PVC support
{{- if .Values.notebooks.volumes }}
  volumes:
    - name: notebooks-dir
  {{- if .Values.notebooks.volumes.hostPath }}
      hostPath:
        path: {{ .Values.notebooks.volumes.hostPath }}
  {{- end }}
{{- end }}